cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -47,6 +58,42 @@ export default {
     return cy.do(keywordInput.fillIn(kw));
   },
 
+  selectCallNumberBrowse() {
+    // cypress can't draw selected option without wait
+    cy.wait(1000);
+    return cy.do(Select('Search field index').choose('Browse call numbers'));
```

Comment:
why do you add this return statement?

Reply:
I can use it with promise like

`selectCallNumberBrowse.then(element => something)`

I think it is not obligatory, but it can be useful

Reply:
I prefer to keep code related with Present. Future is not predictable. If you sure that you will use it - you are right